docs/public/schema.json

```diff
@@ -1 +1,252 @@
\ No newline at end of file
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
Minor thing, but if we're going to format a codegen file we should do the formatting as part of the codegen.

Reply:
Ah yeah this got accidentally formatted. Reverting